www_src/components/snackbar/snackbar.less

```diff
@@ -0,0 +1,17 @@
+.snackbar {
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: #444;
+  line-height: 3em;
+  text-align: center;
+  color: #EEE;
+  height: 3em;
```

Comment:
I'd animate positioning in that case. You may need some JS to accomplish this, but I can pretty much guarantee it will break with fixed height.